projects/demo/src/app/app.component.html

```diff
@@ -1,106 +1,74 @@
+  <h1 class="navbar-brand mb-0">ngx-valdemort <small class="d-none d-sm-inline">by Ninja Squad</small></h1>
+  <button class="navbar-toggler" type="button" aria-controls="navbarContent" aria-expanded="false" aria-label="Toggle navigation" (click)="navbarCollapsed = !navbarCollapsed">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="navbar-collapse" id="navbarContent" [class.collapse]="navbarCollapsed">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="documentation/index.html">API documentation</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="https://github.com/Ninja-Squad/ngx-valdemort">
+          <img alt="" src="/assets/github.png" />
+          Github
+        </a>
+      </li>
+    </ul>
+  </div>
+</nav>
 
-    <div class="form-group">
-      <label for="lastName">Last name (required, min length 2, only letters -> custom message)</label>
-      <input id="lastName" formControlName="lastName" class="form-control" />
-      <val-errors controlName="lastName" label="The last name">
-        <ng-template valError="pattern">The last name must only contain letters</ng-template>
-      </val-errors>
-    </div>
+<main role="main" class="container mt-5">
 
-    <div class="form-group">
-      <label for="age">Age (required, min 18 -> custom message with error exposed)</label>
-      <input id="age" formControlName="age" type="number" class="form-control"/>
-      <val-errors controlName="age" label="The age">
-        <ng-template valError="min" let-error="error">You must be at least {{ error.min | number }} years old</ng-template>
-      </val-errors>
+  <div class="jumbotron d-sm-flex">
+    <div class="logo">
+      <img src="/assets/logo.svg" alt="logo" />
     </div>
-
-    <div formGroupName="credentials">
-      <div class="form-group">
-        <label for="password">Password (required)</label>
-        <input id="password" formControlName="password" class="form-control" type="password"/>
-        <val-errors controlName="password" label="The password"></val-errors>
-      </div>
-
-      <div class="form-group">
-        <label for="password">Password confirmation (required)</label>
-        <input id="confirmation" formControlName="confirmation" class="form-control" type="password"/>
-        <val-errors controlName="confirmation" label="The password confirmation"></val-errors>
-        <val-errors [control]="form.get('credentials')">
-          <ng-template valError="validCredentials">The confirmation <strong>must</strong> match the password</ng-template>
-        </val-errors>
-      </div>
+    <div class="flex-fill">
+      <h2>ngx&#8209;valdemort <small>by&nbsp;Ninja&nbsp;Squad</small></h2>
+      <p class="lead">
+        Simple, consistent validation error messages for your Angular forms.
+      </p>
     </div>
+  </div>
 
-    <button class="btn btn-primary">Submit</button>
+  <h2>The problem</h2>
```

Comment:
Something a little catchier maybe? Along the lines of `Tired of writing the same validation logic again, and again?`?